modules/client/shared/src/main/scala/com/azavea/stac4s/api/client/StacClient.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.azavea.stac4s.api.client
+
+import com.azavea.stac4s._
+
+import eu.timepit.refined.types.string.NonEmptyString
+
+/** TODO: instead of returning F[List] we can return fs2.Stream */
```

Comment:
Can you open an issue for this that explains why it's a TODO instead of leaving a TODO in the code?

Reply:
https://github.com/azavea/stac4s/issues/198